app/lib/user_attributes.rb

```diff
@@ -40,6 +41,24 @@ def errors
         invalid_keys << "is_stored_locally"
       end
 
+      permissions = config["permissions"]
+      if permissions
+        missing_keys.concat(PERMISSION_KEYS.reject { |key| permissions.keys.include? key }.map { |key| "permissions.#{key}" })
+        unknown_keys.concat(permissions.keys.reject { |key| PERMISSION_KEYS.include? key }.map { |key| "permissions.#{key}" })
+
+        non_integral_values = permissions.keys.reject { |key| permissions[key].is_a? Integer }.map { |key| "permissions.#{key}" }
```

Comment:
Humm.. is there a clearer name than this?
I think it's the most correct, but it took me a moment to realise this was integral as in integer and not as in integrated

Reply:
How about `non_integer_values`?

Reply:
yeah that works for me!